lib/WP_Auth0_LoginManager.php

```diff
@@ -232,9 +232,13 @@ public function redirect_login() {
 			throw new WP_Auth0_LoginFlowValidationException( $e_message, $e_code );
 		}
 
+		$access_token = $data->access_token;
+		$id_token = $data->id_token;
```

Comment:
how can you be so sure that the `id_token` is present?? you might want to null check like you do for refresh tokens.

Reply:
only if the `response_type` includes `id_token` (i.e. `token id_token`) the `id_token` should be present. At least that's how it's supposed to work :D

Reply:
Well, this being more of an app than an SDK, that's something the plugin controls and doesn't change. Both flows currently get an id_token and have no reason to change.

Reply:
Reading this again after yesterday's discussion, this method handles code exchange redirection so here we have access_token and id_token. No need to null-check them 👍

Reply:
I confirmed that OIDC and non-OIDC applications both prove `access_token` and `id_token` with `response_type=code`